### PR TITLE
y & u out, if for CD demand, add WiNDC nat test

### PR DESCRIPTION
**Pull request by EliLazarus**
This includes the 2 fixes needed to correctly solve the WiNDC national model, both with the 'original' data, and the secondard Julia-generated WiNDCdata.

1) The y_over_y_bar and u_over_u_bar functions and their inclusion in the compensated demand, supply, and final demand equations (and their sub-expression) were removed. In our construction, those variables, the indexing for those appear in the final equations.
2) The Cobb-Douglas case (demand elasticity==1) in the expenditure expression has a new conditional branch. 1/(1-elasticity) as was being constructed by just using the CES form caused an Inf. Somehow that was getting through until something about the Julia-generate WiNDC data made it evaluate in a different way.

I also add the WiNDC national model with 2 additional demand elasticity value counterfactuals as a test.

### CODE REVIEW

**Comment by EliLazarus** [comment]
The breaking test is (pretty sure) some issue with the JLD2 data files used in the WiNDC national model test. 
I thought the issue was fixed by down-versioning to limit to JLD2 0.4.44 which had fixed the same error for me locally at some point, but it doesn't seem to fix with that added to the Project.toml, and I then couldn't replicate that bug locally.

The JLD2 has a Dict of DenseAxisArrays, which I didn't manage to transfer to csvs or xlsx.

**Comment by EliLazarus** [comment]
[Suggested fix from JLD2](https://github.com/JuliaIO/JLD2.jl/issues/549#issuecomment-1949928224)

**Comment by EliLazarus** [comment]
It's (surely) the PATH license! (but how to add it...)

File: test/test_WiNDC_national.jl
Comment (mitchphillipson) [inline test/test_WiNDC_national.jl:1]: I may come back to this file and clean it up. The model definition can be more concise and there must be a better way to  run all the tests. Not a blocking issue though.

**Review by mitchphillipson — APPROVED** [review]
Mainly CSV files and tests. Major change is removing y/\bar{y}.

File: test/test_WiNDC_national.jl
Comment (EliLazarus) [reply test/test_WiNDC_national.jl:1]: Sounds good. We also have it on the list to develop a nicer API in general, to make it more user-friendly, which to me means something like intuitive for an economics student without much coding background.
Re the tests, I could run a loop or something, but as is I generate the code with Excel and copy paste which isn't bad, and I don't think the text repetition affects the processing, so I kind of like how it's all easily readable and clear for anyone who might look at it.

**Pull request merged**